draft-ietf-ohai-ohttp.md

```diff
@@ -652,8 +652,10 @@ fields that are listed in the Connection header field.
 The oblivious proxy resource interacts with the oblivious request resource by
 constructing a request using the same restrictions as the client request, except
 that the target URI is the oblivious request resource.  The content of this
-request is copied from the client.  The oblivious proxy resource MUST NOT add
-information about the client to this request.
+request is copied from the client.  The oblivious proxy resource SHOULD NOT add
+information about the client to this request, though some deployments or
+application use cases may warrant additional information; see
+{{proxy-responsibilities}} for more information.
```

Comment:
Should we instead look to do "MUST NOT ... except" with a very specific carve-out?  It seems like a "SHOULD NOT" opens the door a little too wide.

If this is just shadow bans, then we're golden.  But there's a thin end of a wedge here.  Is it OK to do the equivalent of DNS client subnet?  What about User-Agent?

Reply:
I reverted this to MUST NOT and carved out the exceptional case.